package.json

```diff
@@ -30,7 +30,7 @@
     "prettier": "^1.1.0"
   },
   "dependencies": {
-    "chunk-manifest-webpack-plugin": "~1.0.0"
+    "chunk-manifest-webpack-plugin": "~1.1.1"
```

Comment:
Might as well patch to 1.1.2 as there apparently was a regression bug in 1.1.1. See below:
https://github.com/soundcloud/chunk-manifest-webpack-plugin/commit/1321345aa97c8f944f6f8f7cf3e3dbfff5eea31b

So I'd suggest `~1.1.2`.